src/Components/Alert/Hooks/useAlertTracking.ts

```diff
@@ -55,5 +59,28 @@ export const useAlertTracking = () => {
 
       trackEvent(payload)
     },
+
+    clickedAddFilters: () => {
+      const payload: ClickedAddFilters = {
+        action: ActionType.clickedAddFilters,
+        context_module: ContextModule.alertFilters,
+      }
+
+      trackEvent(payload)
+    },
+
+    clickedArtworkGroup: (internalId: string, slug: string) => {
+      const payload: ClickedArtworkGroup = {
+        action: ActionType.clickedArtworkGroup,
+        context_module: ContextModule.alertConfirmation,
+        context_page_owner_type: OwnerType.alertConfirmation,
```

Comment:
**suggestion**: We can use the values from the `useAnalyticsContext` hook further up in this file. Those values should always match the page we're on.

```
context_page_owner_type: contextPageOwnerType,
```